pyrq/unique_queues.py

```diff
@@ -66,29 +69,32 @@ def get_count(self) -> int:
     def add_item(self, item) -> bool:
         """
         :param item: Anything that is convertible to str
-        :return: Returns true if item was inserted into queue, false otherwise
         """
-        result = self.redis.lpush(self.name, item)
+        self.add_command(keys=[self.name, self.set_queue_name], args=[str(item)])
```

Comment:
`name` je vlastně `queue_name` a `set_queue_name` žádná fronta není, je to `set` :)